ui/raidboss/data/05-shb/dungeon/matoyas_relict.js

```diff
@@ -0,0 +1,325 @@
+      delaySeconds: 3,
+      durationSeconds: 6,
+      infoText: (data, _, output) => output.stepIn(),
+      outputStrings: {
+        stepIn: {
+          en: 'Step in Puddle near the Cloud',
+          ja: '雲に近い水を踏む',
+          cn: '站在靠近云朵的水流里等待浮空',
+        },
+      },
+    },
+    {
+      id: 'Matoyas Porxie Tender Loin',
+      netRegex: NetRegexes.startsUsing({ id: '5913', source: 'Mother Porxie', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '5913', source: 'Muttersau', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '5913', source: 'mère porxie', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '5913', source: 'マザーポークシー', capture: false }),
+      condition: Conditions.caresAboutAOE(),
+      delaySeconds: 3,
+      response: Responses.aoe(),
+    },
+    {
+      id: 'Matoyas Porxie Huff and Puff',
+      netRegex: NetRegexes.startsUsing({ id: '5919', source: 'Mother Porxie', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '5919', source: 'Muttersau', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '5919', source: 'mère porxie', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '5919', source: 'マザーポークシー', capture: false }),
+      alertText: (data, _, output) => output.getKnocked(),
+      outputStrings: {
+        getKnocked: {
+          en: 'Get Knocked into Safe (no anti-knockback)',
+          ja: 'ボスの正面に (堅実魔効かない)',
+          cn: '站在Boss正面 (防击退无效)',
+        },
+      },
+    },
+    {
+      id: 'Matoyas Porxie Meat Mallet',
+      netRegex: NetRegexes.startsUsing({ id: '5916', source: 'Mother Porxie', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '5916', source: 'Muttersau', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '5916', source: 'mère porxie', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '5916', source: 'マザーポークシー', capture: false }),
+      alertText: (data, _, output) => output.awayFromAoe(),
+      outputStrings: {
+        awayFromAoe: {
+          en: 'Away From AoE',
+          ja: '反対側へ',
+          cn: '对面躲避坠落',
+        },
+      },
+    },
+    {
+      id: 'Matoyas Porxie Barbeque',
+      netRegex: NetRegexes.ability({ id: '5916', source: 'Mother Porxie', capture: false }),
+      netRegexDe: NetRegexes.ability({ id: '5916', source: 'Muttersau', capture: false }),
+      netRegexFr: NetRegexes.ability({ id: '5916', source: 'mère porxie', capture: false }),
+      netRegexJa: NetRegexes.ability({ id: '5916', source: 'マザーポークシー', capture: false }),
+      delaySeconds: 5,
+      alarmText: (data, _, output) => output.awayFrom(),
+      outputStrings: {
+        awayFrom: {
+          en: 'Away From Boss',
+          ja: 'ボスに離れ',
+          cn: '远离Boss所在的行',
+        },
+      },
+    },
+    {
+      id: 'Matoyas Porxie Sucked In',
+      netRegex: NetRegexes.gainsEffect({ effectId: '5916', capture: false }),
```

Comment:
```suggestion
      netRegex: NetRegexes.gainsEffect({ effectId: '9B6', capture: false }),
```

`26|2020-12-09T23:54:40.1260000-08:00|9b6|Sucked In|30.00|E0000000||12345678|Tini Poutini|00|184168|||aa5ae87156e3d07d84b63ced15f925fe`

Reply:
Oops, maybe I confused with the skill id, thanks